### DISCUSSION TITLE
Guidance on topic model selection

### DISCUSSION THREAD

**Discussion by AmosFong1** [aertslab/pycisTopic#153] [Q&A]
Similar to #114.

Loglikelihood does not seem very informative as it is not optimized even at iteration 500. In the [vignette](https://pycistopic.readthedocs.io/en/latest/notebooks/human_cerebellum.html#Model-selection), loglikelihood and minmo (topic-coherence) are the recommended metrics for model selection. What are your thoughts on this result, and is loglikelihood still informative in this case?

![topics](https://github.com/user-attachments/assets/f6c014a4-61a3-4524-b473-3da11f3d2626)

**Comment by SeppeDeWinter** [comment]
Hi @AmosFong1 

Yes, the selection is not always 100% objective.
What can help is to plot each metric individually. In this plot all metrics are scaled between 0 and 1.
However, the absolute difference between each iteration for some metrics might not be as large as the 
relative difference you are plotting here.

I hope this helps?

All the best,

Seppe
